data/transition-sites/gds_alphagov_alert.yml

```diff
@@ -0,0 +1,10 @@
+---
+site: gds_alphagov_alert
+whitehall_slug: government-digital-service
+homepage: https://alert.publishing.service.gov.uk/
```

Comment:
@alexmuller sent me the following link, which explain the reason:
https://github.com/alphagov/transition-config/pull/1111#discussion_r34457874